md/migrate-a-form-from-6-x.md

```diff
@@ -18,3 +18,8 @@ You can do so with a process that uses a connector to retrieve external data and
 6. In the Studio, at pool level and for each human task, go to the **_Details_** panel, **_Execution_** tab, **_Form_** section, and change the option from **Legacy (6.x)** to **_UI Designer_**. You can also generate them in another development environment. In this case, choose **_External URL_** option.
 
 When you export a diagram, forms created in the UI Designer are embedded by default in the .bos and .bar files, to share between Bonita Studios (.bos) or install in a Bonita Portal (.bar), so you don't need to manually export them and import them one by one.
+
+7. Confirmation page. After a 6.x form was submitted, the Portal handled a Confirmation page that browsed for the next available task in the current case for the logged user. In case there was no such a subsequent task for the user, she needed to go back to the Task List with a click, which happened most of the time. 
```

Comment:
This is not the case since 7.3 and the new task list. This is not a change specific to 7.8

Reply:
Maybe: "Before 7.3, after a 6.x form was submitted, the Portal used to handle"...

Reply:
I think to keep the documentation easier to read and as we no longer support 7.3 we can leave that information out of this documentation page.